docs/contributing/guidelines/workflow.md

```diff
@@ -96,7 +96,6 @@ The following template is automatically provided when you raise a pull request a
     ## Reviewers (*optional*)
 
     ----------------------------------------------------------------------------------------------------------------
-    ## Release Notes (*required for feature/major PRs*)
```

Comment:
not sure why this is being removed from the template? the explanatory text further down still has a section on release notes

Reply:
We removed release notes section from the PR template. the description is used as release notes.